src/Microsoft.Framework.Runtime.Hosting/Dependencies/FrameworkReferenceResolver.cs

```diff
@@ -42,6 +35,17 @@ public bool TryGetAssembly(string name, NuGetFramework targetFramework, out stri
             path = null;
             version = null;
 
+            if(!targetFramework.IsDesktop())
```

Comment:
This class is just a nice model over the `C:\Program Files (x86)\Reference Assemblies\Microsoft`